frontend/app/templates/components/forms/signup-form.hbs

```diff
@@ -22,8 +22,12 @@
             <div class="field required">
                 <div class="ui left icon input">
                     <i class="lock icon"></i>
-                    {{input type="password" value=password name="password" placeholder="Password"}}
+                    {{input type="password" value=password  name="password"  id="pass" placeholder="Password"}}
                 </div>
+                <meter  id="password-strength-meter"  value={{level}} ></meter>
+
```

Comment:
please remove the extra lines from the file.
then we can merge this into the branch
cheers! thanks!